lib/util/problem.js

```diff
@@ -62,6 +62,9 @@ const problems = {
     // { field: "the name of the extraneous parameter" }
     unexpectedAttribute: problem(400.4, ({ field }) => `Passed parameter ${field} was not expected.`),
 
+    // { fields: "the name of the extraneous parameters" }
+    unexpectedAttributes: problem(400.5, ({ fields }) => `Passed parameters (${fields.join(', ')}) were not expected.`),
```

Comment:
We previously used 400.5 for something else. It was removed in #191. I think it'd be a good idea to add a new Problem code rather than reusing the old one.